roles/junit2json/tasks/convert.yml

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# tasks file for redhatci.ocp.junit2json
+# task list: convert.yml converts xml into JSON
+
+- name: Read file content
+  ansible.builtin.set_fact:
+    junit2json_xml_report_content: "{{ lookup('ansible.builtin.file', xml_report) }}"
+
+- name: Generate content hash from the content
+  ansible.builtin.set_fact:
+    junit2json_xml_report_hash_path: "{{ xml_report }}.{{ junit2json_hash }}"
+    junit2json_xml_report_hash_curr: "{{ junit2json_xml_report_content | ansible.builtin.hash(junit2json_hash) }}"
+    junit2json_xml_report_hash_old: ""
```

Comment:
Do we need it here if it's empty?

Reply:
why is this conversation resolved? I don't see an answer